client/vmmVSwitchPolicyCont_service.go

```diff
@@ -0,0 +1,331 @@
+	dn := fmt.Sprintf("%s/rsvswitchOverrideStpPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideStpPol", dn, StpIfPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideStpPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideStpPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideStpPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideLldpIfPol(parentDn, LldpIfPolName string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideLldpIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideLldpIfPol", dn, LldpIfPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideLldpIfPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideLldpIfPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideLldpIfPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideMcpIfPol(parentDn, tnMcpIfPolName string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideMcpIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
+			}
+		}
+	}`, "vmmRsVswitchOverrideMcpIfPol", dn, tnMcpIfPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchOverrideMcpIfPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideMcpIfPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchOverrideMcpIfPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideCdpIfPol(parentDn, CdpIfPolName string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideCdpIfPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
```

Comment:
We usually add "annotation" parameter in the attributes payload with value as "orchestrator:terraform" if relation class support annotation.